### PR TITLE
Setting defaults logs

### PR DESCRIPTION
**Pull request by otherview**
### Why is this change needed?

- Part of https://github.com/obscuronet/obscuro-internal/issues/870
- Fixes the default logs settings.

### What changes were made as part of this PR:

- Provide a high level list of the changes made

### What are the key areas to look at

- Describe the key areas for a reviewer to look at 

### :rotating_light: Definition of done :rotating_light:
- [ ] Unit tests added to cover new or changed functionality 
- [ ] Docs pages updated to cover new or changed functionality
- [ ] [Changelog.md](https://github.com/obscuronet/go-obscuro/blob/main/docs/testnet/changelog.md) updated

### CODE REVIEW

File: go/common/log/log.go [hunk 1]
```diff
@@ -64,6 +66,8 @@ func Panic(msg string, args ...interface{}) {
 func ParseLevel(levelStr string) zerolog.Level {
 	lvl, err := zerolog.ParseLevel(levelStr)
 	if err != nil {
+		// we purposefully
```
Comment (joeldudley) [inline go/common/log/log.go:69]: Missing text?

**Review by joeldudley — APPROVED** [review]
LGTM

File: go/common/log/log.go (hunk 1 above)
Comment (otherview) [reply go/common/log/log.go:69]: Yes, thank you ! :)

**Comment by tudor-malene** [comment]
@joeldudley , do you know what this pr is about? Do we need to merge it?

**Comment by joeldudley** [comment]
@tudor-malene Not sure about this one. Don't know if Pedro ended up addressing this in another way.

**Comment by otherview** [comment]
Closing this, superseeded by https://github.com/obscuronet/go-obscuro/pull/740

**Pull request closed without merging**